app/views/messages/index.html.erb

```diff
@@ -23,13 +23,31 @@
 
               <% if contact_record.contact_record_type == :outgoing_text_message %>
                 <span class="contact-record__phone_number">
-                  Text to <%= contact_record.client.formatted_phone_number %>
+                  Text to <%= local_phone_number(contact_record.client.phone_number) %>
```

Comment:
IMHO we should store the phone number we sent it to, at the time we send it, so we have a record of what happened, even if the client's phone number changed.

I'll probably make that change, just logging it here.